tools/lsp/preview/native.rs

```diff
@@ -177,6 +180,10 @@ pub(super) fn open_ui_impl(preview_state: &mut PreviewState) -> Result<(), slint
         cache.ui_is_visible = false;
         slint::CloseRequestResponse::HideWindow
     });
+    if matches!(window_behaviour, WindowBehavior::BringWindowToFront) {
+        use i_slint_backend_winit::WinitWindowAccessor;
+        ui.window().with_winit_window(|winit_window| winit_window.focus_window());
```

Comment:
(unfortunate to have to use private API, but i understand you want to get a quick fix rather than adding new API that our user could also use)

Reply:
I mean, if `Window::focus_window()` is a no-brainer for you, then I can add it :). Part of the issue I can see is that it's once again not supported on Wayland heh.